opentelemetry-sdk/src/opentelemetry/sdk/_metrics/metric_reader_storage.py

```diff
@@ -34,7 +40,7 @@ class MetricReaderStorage:
     def __init__(self, sdk_config: SdkConfiguration) -> None:
         self._lock = RLock()
         self._sdk_config = sdk_config
-        self._view_instrument_match: Dict[
+        self._instrument_view_instrument_matches: Dict[
```

Comment:
I actually found it hard to read when we were using `self._view_instrument_matches` because that private attribute is not a sequence of `_ViewInstrumentMatch` objects. I find `_instrument_view_instrument_matches` a more descriptive name since it is a dictionary where a key is an instrument and a value is a sequence of `_ViewInstrumentMatch` objects.

Just my opinion, of course.

Reply:
+1 on name being difficult to read.